Modules/Core/Transform/include/itkMatrixOffsetTransformBase.hxx

```diff
@@ -103,7 +105,7 @@ MatrixOffsetTransformBase<TParametersValueType, VInputDimension, VOutputDimensio
   m_Singular = false;
   m_InverseMatrix.SetIdentity();
   m_InverseMatrixMTime = m_MatrixMTime;
-  this->Modified();
+  this->SetFixedParameters(tmp_fixed_parameters);
 }
```

Comment:
Are you sure it is no longer necessary to call `this->Modified()` here? Just wondering 🤷

Reply:
Good catch!  That was a mistake!